ui/apps/platform/cypress/integration/integrations/notifiers.test.js

```diff
@@ -59,7 +59,18 @@ describe('Notifier Integrations', () => {
             cy.get(selectors.buttons.test).should('be.disabled');
             cy.get(selectors.buttons.save).should('be.disabled');
 
-            // Step 2, check fields for invalid formats
+            // Step 2, check conditional fields
+
+            // Step 2.1, enable container IAM role, this should remove the AWS credentials fields
+            getInputByLabel('Use container IAM role').click();
+            cy.get('[id=notifier.awsSecurityHub.credentials.accessKeyId]').should('not.exist');
+            cy.get('[id=notifier.awsSecurityHub.credentials.secretAccessKey]').should('not.exist');
+            // Step 2.2, disable container IAM role, this should render the AWS credentials fields again
+            getInputByLabel('Use container IAM role').clear();
```

Comment:
The method for check box is `click()` again to clear it. The `clear()` method is for text content.

Ditto for `'Use workload identity'` below.